src/multivariate/mvdiscretenonparametric.jl

```diff
@@ -0,0 +1,55 @@
+const MvDiscreteNonParametric{
+    T<:AbstractVector{<:Real},
+    P<:Real,
+    Ts<:AbstractVector{T},
+    Ps<:AbstractVector{P},
+} = GeneralDiscreteNonParametric{Multivariate,T,P,Ts,Ps}
```

Comment:
What's the motivation for introducing this type-alias? It seems we could work with `GeneralDiscreteNonParametric` directly? Also it would only cover the multivariate case even though other `ArrayLikeVariate` cases could be handled in the same way.

I suggest removing this file completely and moving the definitions to `src/generaldiscretenonparametric.jl` instead (it should not be in the `multivariate` subfolder since it is not necessarily a multivariate distribution).

Reply:
Do we just drop the `MvDiscreteNonParametric` then? Should I change all for `GeneralNonParametric`?